<launch>
    <include file="$(find rs_robot_sensors)/launch/rs_d435_t265.launch"/>

    <node pkg="tf2_ros" type="static_transform_publisher" name="odom_to_t265_odom" args="0.15 0 0.12 0 0 0 /odom /t265_odom_frame"/>
    <!--
    <node pkg="tf2_ros" type="static_transform_publisher" name="t265_link_to_roomba" args="-0.15 0 -0.12 0 0 0 /t265_pose_frame /base_footprint"/>
    -->
    <node pkg="rs_robot_sensors" type="robot_root_transform_publisher" name="t265_pose_to_base_footprint">
        <param arg="parent_frame" value="t265_pose_frame"/>
        <param arg="child_frame" value="base_footprint"/>
    </node>
    <node pkg="tf2_ros" type="static_transform_publisher" name="t265_to_d435" args="-0.03 0 0.03 0 0 0 /t265_pose_frame /d435_link"/>
<!--
    <node pkg="tf" type="static_transform_publisher" name="odom_to_t265_odom" args="0 0 0 0 0 0 /odom /t265_odom_frame 100"/>
    <node pkg="tf" type="static_transform_publisher" name="t265_link_to_roomba" args="-0.15 0 -0.12 0 0 0 /t265_pose_frame /base_footprint 100"/>
    <node pkg="tf" type="static_transform_publisher" name="t265_to_d435" args="-0.03 0 0.03 0 0 0 /t265_pose_frame /d435_link 100"/>
-->

</launch>